code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -239,15 +237,15 @@ var/list/blacklisted_pylon_turfs = typecacheof(list(
 
 
 
-/obj/structure/cult/culttome
+/obj/structure/cult/itemspawner/tome
 	name = "archives"
 	desc = "A desk covered in arcane manuscripts and tomes in unknown languages. Looking at the text makes your skin crawl."
 	icon_state = "tomealtar"
 	health = 125 //Slightly sturdy
 	death_message = "<span class='warning'>The desk breaks apart, its books falling to the floor.</span>"
 	death_sound = 'sound/effects/wood_break.ogg'
 
-/obj/structure/cult/culttome/attack_hand(mob/living/user)
+/obj/structure/cult/itemspawner/tome/attack_hand(mob/living/user)
```

Comment:
`attack_hand` can be all defined on the `/obj/structure/cult/itemspawner` level, removing a bunch of duplicated code